### PR TITLE
FORMS-11692 : Upgrade af2-web-runtime version and add Rule editor UI end to end test cases in Forms Core Components

### PR DESCRIPTION
**Pull request by arunattri26**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/libs/commons/guideSelectors.js [hunk 1]
```diff
@@ -170,7 +170,10 @@ var selectors = {
             editRule : "#EditableToolbar [data-action='editexpression']",
             createRuleButton : "#create-rule-button",
             saveRule : ".exp-Save-Button",
-            closeRuleEditor : ".exp-Close-Button"
+            closeRuleEditor : ".exp-Close-Button",
+            sideToggleButton : "#exp-toggle-sidepanel-button",
+            formObjectsTab : ".exp-sidepanel coral-tabview coral-tab coral-tab-label:first",
```
Comment (rismehta) [inline ui.tests/test-module/libs/commons/guideSelectors.js:175]: Is it possible to have a specific selector without having coral dependency ?

File: ui.tests/test-module/specs/ruleeditor/ruleEditor.authoring.spec.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
+const commons = require('../../libs/commons/commons'),
+    sitesSelectors = require('../../libs/commons/sitesSelectors'),
+    formsSelectors = require('../../libs/commons/guideSelectors'),
+    afConstants = require('../../libs/commons/formsConstants');
+
+describe('Rule editor sanity for core-components',function(){
```
Comment (rismehta) [inline ui.tests/test-module/specs/ruleeditor/ruleEditor.authoring.spec.js:6]: Can we also add these test cases in SITE page, you can refer other test cases, we follow this pattern everywhere

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.tests/test-module/libs/commons/guideSelectors.js (hunk 1 above)
Comment (arunattri26) [reply ui.tests/test-module/libs/commons/guideSelectors.js:175]: There is no specific selector available for the tabs currently. So we will have to depend upon the coral tag.

File: ui.tests/test-module/specs/ruleeditor/ruleEditor.authoring.spec.js (hunk 2 above)
Comment (arunattri26) [reply ui.tests/test-module/specs/ruleeditor/ruleEditor.authoring.spec.js]: Done.

**Comment by arunattri26** [comment]
moved to  new PR: https://github.com/adobe/aem-core-forms-components/pull/1117

**Pull request closed without merging**